types/node/ts3.1/tsconfig.json

```diff
@@ -7,12 +7,11 @@
         "module": "commonjs",
         "target": "es6",
         "lib": [
-            "es6",
-            "dom"
+            "es6"
```

Comment:
shouldn't we keep "dom" here to ensure that node and dom are compatible?

Reply:
Yes and no.
I don't think they are going to be compatible in the long run and it's already causing issues (see TextEncoder/console etc.), eventually these will have to be broken up or `console` etc. need to be moved to a shared lib.

I will restore it for now but I still think it's a nasty anti pattern.

Reply:
I' assume that a lot people don't do this on purpose; it just happens because of dependencies.